src/backend/oneapi/resize.cpp

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
This file and reorder.cpp have changed permissions to 100755. 
revert to 100644. 
and fix reorder.cpp missing (?)

Reply:
and check other files for permissions, some mismatching